ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/dot1x.md

```diff
@@ -111,4 +125,44 @@
 
         # Interval period in seconds.
         accounting_update_interval: <int; 5-65535>
+
+      # Web authentication feature authenticates a supplicant through a web page, referred to as a captive portal.
+      captive_portal:
```

Comment:
There is an option of start and we can only have captive-portal command. 
```
wan1-dc1(config-dot1x)#captive-portal ?
  access-list  Configure access control list
  ssl          Configure SSL related options
  start        Configure captive-portal start parameters
  url          Configure captive portal URL
  <cr>         
```

Reply:
May be `start` is in other version, I can see only these-
```
s1-leaf2(config-dot1x)#captive-portal ?
  access-list  Configure access control list
  ssl          Configure SSL related options
  url          Configure captive portal URL
  <cr>  
```

Can you please share the config also for `start`

Reply:
Thanks for sharing the config, added both keys